app/views/liquids/_asciinema.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="ltag_asciinema">
+  <iframe frameborder="0"
+    scrolling="no"
+    id="asciinema_<%= id %>"
+    src="https://asciinema.org/a/<%= id %>"
```

Comment:
I think the `src` is missing `/iframe` at the end.
```suggestion
    src="https://asciinema.org/a/<%= id %>/iframe"
```

Reply:
Huh, yeah, serves me right for doing this past midnight. Thanks!